installer/ubuntu/bootstrap

```diff
@@ -15,9 +15,10 @@ if true || ! wget -O- --no-verbose --timeout=60 -t2 "$d"  \
         | tar -C "$tmp" --strip-components=1 -zx 2>/dev/null; then
     echo 'Download from Debian gitweb failed. Trying latest release...' 1>&2
     d='http://httpredir.debian.org/debian/pool/main/d/debootstrap/'
-    f="`wget -O- --no-verbose --timeout=60 -t2 "$d" \
-            | sed -ne 's ^.*\(debootstrap_[0-9.]*.tar.gz\).*$ \1 p' \
-            | tail -n 1`"
+    d_api='https://sources.debian.net/api/src/debootstrap/'
+    f="`wget -O- --no-verbose --timeout=60 -t2 "$d_api" \
+            | sed -ne 's ^.*\"version\":\ \"\([0-9.]*\)\".*$ debootstrap_\1.tar.gz p' \
```

Comment:
Yes, your suggestions seem to work; and good not to rely too much on the current api representation; fixed in the new commit